test/benchmark/shell_scripts/iGenVar_init.sh

```diff
@@ -0,0 +1,51 @@
+# -------- -------- Prepare logfile and runtime computation -------- -------- #
+mkdir logs
+echo "Logifle written to: initial_steps.log"
+exec 3>&1 1>>logs/initial_steps.log 2>&1                                # write stdout 1,2 to logfile 3 to console
+echo $(date)
+start=$(date +%s)                                                       # get starting date
+
+# -------- -------- get & build iGenVar -------- -------- #
+mkdir Repos
+cd Repos/
+git clone https://github.com/seqan/iGenVar.git
+wait
+cd iGenVar
+git submodule update --recursive --init
+wait
+echo "$(tput setaf 1)$(tput setab 7)------- iGenVar downloaded (1/4) --------$(tput sgr 0)" 1>&3
+
+cd ../..
+mkdir build && cd build && mkdir iGenVar && cd iGenVar
+cmake ../../Repos/iGenVar/ -Weverything
+wait
+make -j 16
+wait
+make test && make doc
+wait
+echo "$(tput setaf 1)$(tput setab 7)------- iGenVar builded (2/4) --------$(tput sgr 0)" 1>&3
+
+# -------- -------- get data and unzip -------- -------- #
+cd ../..
+mkdir data && cd data
+
+mkdir short_reads && cd short_reads
+cd ..
+echo "$(tput setaf 1)$(tput setab 7)------- short reads downloaded (3/4) --------$(tput sgr 0)" 1>&3
+
+mkdir long_reads && cd long_reads
+if wget https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/data/AshkenazimTrio/HG002_NA24385_son/PacBio_CCS_10kb/alignment/HG002.Sequel.10kb.pbmm2.hs37d5.whatshap.haplotag.RTG.10x.trio.bam -nv; then echo "PacBio CSS 10kb download succesful"; else echo "PacBio CSS 10kb download-link broken" 1>&3; exit 1; fi
```

Comment:
```suggestion
wget --retry-connrefused --waitretry=30 --read-timeout=30 --timeout=30 --tries=20 --no-clobber --no-verbose https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/data/AshkenazimTrio/HG002_NA24385_son/PacBio_CCS_10kb/alignment/HG002.Sequel.10kb.pbmm2.hs37d5.whatshap.haplotag.RTG.10x.trio.bam
```

the options make `wget` more robust. with `set -ex` you won't need to check for the return code since `-ex` will take care of this